From 19712b6c7e1758aa82f8eb5ed0cb9acac4f7dd76 Mon Sep 17 00:00:00 2001
From: "dongwan0605.kim" <dongwan0605.kim@lge.com>
Date: Sun, 6 May 2018 18:04:52 +0900
Subject: [PATCH 11/13] Update text size of overflow number view

When density or font size is updated, overflow number view's size
was not updated.

Test: Group notification with overflow number existing
> Change density or font size > Check overflow number view's size

Change-Id: I6f464b12f83b954f7fbfdd5d64153598b5407e08
---
 .../statusbar/notification/HybridGroupManager.java    | 11 +++++++----
 .../stack/NotificationChildrenContainer.java          |  1 +
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/notification/HybridGroupManager.java b/packages/SystemUI/src/com/android/systemui/statusbar/notification/HybridGroupManager.java
index 3ed8cce21ac..3975ec2d277 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/notification/HybridGroupManager.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/notification/HybridGroupManager.java
@@ -37,10 +37,10 @@ public class HybridGroupManager {
     private final NotificationDozeHelper mDozer;
     private final ViewGroup mParent;
 
-    private final float mOverflowNumberSizeDark;
-    private final int mOverflowNumberPaddingDark;
-    private final float mOverflowNumberSize;
-    private final int mOverflowNumberPadding;
+    private float mOverflowNumberSizeDark;
+    private int mOverflowNumberPaddingDark;
+    private float mOverflowNumberSize;
+    private int mOverflowNumberPadding;
 
     private int mOverflowNumberColor;
     private int mOverflowNumberColorDark;
@@ -50,7 +50,10 @@ public class HybridGroupManager {
         mContext = ctx;
         mParent = parent;
         mDozer = new NotificationDozeHelper();
+        initDimens();
+    }
 
+    public void initDimens() {
         Resources res = mContext.getResources();
         mOverflowNumberSize = res.getDimensionPixelSize(
                 R.dimen.group_overflow_number_size);
diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationChildrenContainer.java b/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationChildrenContainer.java
index fe53104ff9c..7c2918aec10 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationChildrenContainer.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationChildrenContainer.java
@@ -1149,6 +1149,7 @@ public class NotificationChildrenContainer extends ViewGroup {
         }
         recreateNotificationHeader(listener);
         initDimens();
+        mHybridGroupManager.initDimens();
         for (int i = 0; i < mDividers.size(); i++) {
             View prevDivider = mDividers.get(i);
             int index = indexOfChild(prevDivider);
-- 
2.17.0

